identity-apps-core/apps/accounts/src/main/webapp/errors/execution_flow_error.jsp

```diff
@@ -159,10 +159,13 @@
                 <p class="portal-tagline-description">
                     <%= errorDescription %>
                 </p>
-                <button class="ui primary basic button"
-                    onclick="location.href='<%= IdentityManagementEndpointUtil.getURLEncodedCallback(registrationPortalURL) %>';">
-                    <%= i18n(resourceBundle, customText, "sign.up.try.again.button") %>
-                </button>
+                <% if (StringUtils.isNotEmpty(registrationPortalURL) && 
+                    StringUtils.isNotEmpty(request.getParameter("PORTAL_URL"))) { %>
```

Comment:
Let's move the `StringUtils.isNotEmpty(request.getParameter("PORTAL_URL"))` check into the registration URL building logic. Use `isBlank` and also check for the string `"null"`.